buildpack/telemetry/metrics.py

```diff
@@ -55,9 +55,37 @@
     ],
 }
 
+
+def get_apm_metrics_filter(env_var_name):
+    """
+    If we use empty string ("") in the filters that we use for statsd registry,
+    it accepts/denies every metric since we use type as `nameStartsWith`.
+    To prevent breaking the functionality because of this, we need to make sure
+    that we pass empty string to the registry filters only if it's intentional.
+    So, we strip the leading and trailing commas. Additionally we remove all
+    the white spaces to prevent any unintentional mistakes.
+    """
+    env_var_value = os.getenv(env_var_name)
+    if env_var_value is None:
+        return []
+    return env_var_value.replace(" ", "").strip(",").split(",")
+
+
 STATSD_REGISTRY = {
     "type": "statsd",
     "settings": {"port": datadog.get_statsd_port()},
+    "filters": [
```

Comment:
Can we log the final filter values only if the user has provided any of the envvars? Would help us debug if the user has entered unexpected patterns.

Reply:
I think `INFO` is fine too, as it should show up only once during startup right? Else we need to ask them to enable `DEBUG` to figure out any issue.